deprecated/alfresco-transformer-base/src/main/java/org/alfresco/transformer/config/MTLSConfig.java

```diff
@@ -120,11 +125,23 @@ private boolean isKeystoreConfigured()
     }
 
     private RestTemplate createRestTemplateWithSslContext(SSLContextBuilder sslContextBuilder) throws NoSuchAlgorithmException, KeyManagementException {
-        SSLContext sslContext = sslContextBuilder.build();
-        SSLConnectionSocketFactory sslContextFactory = hostNameVerificationDisabled ? new SSLConnectionSocketFactory(sslContext, NoopHostnameVerifier.INSTANCE)
-                : new SSLConnectionSocketFactory(sslContext);
+        final SSLConnectionSocketFactoryBuilder sslConnectionSocketFactoryBuilder =
+                SSLConnectionSocketFactoryBuilder.create()
+                        .setSslContext(sslContextBuilder.build())
+                        .setTlsVersions(TLS.V_1_2, TLS.V_1_3);
+        if (hostNameVerificationDisabled) {
+            sslConnectionSocketFactoryBuilder.setHostnameVerifier(NoopHostnameVerifier.INSTANCE);
+        }
+        final SSLConnectionSocketFactory sslConnectionSocketFactory = sslConnectionSocketFactoryBuilder.build();
+
+        final Registry<ConnectionSocketFactory> sslSocketFactoryRegistry =
+                RegistryBuilder.<ConnectionSocketFactory> create()
+                        .register("https", sslConnectionSocketFactory)
+                        .build();
+
+        final BasicHttpClientConnectionManager sslConnectionManager = new BasicHttpClientConnectionManager(sslSocketFactoryRegistry);
```

Comment:
It's either this - basic, or poolable. I don't think we use poolable "perks" in transform-core at all, so I chose a basic option.